charts/airbyte/values.yaml

```diff
@@ -1258,6 +1258,7 @@ postgresql:
   ## image.repository Repository for airbyte-db statefulset
   image:
     repository: airbyte/db
+    tag: dev-aa084c2cea
```

Comment:
It seems like if you specify in the `.env` file the version it may work: https://github.com/airbytehq/airbyte/issues/32448

Reply:
Modifying the `.env` file works for the Docker deployment, not the k8s one. There's probably different ways of doing this.